pkgs/by-name/aw/awf/package.nix

```diff
@@ -29,23 +32,54 @@ stdenv.mkDerivation rec {
   buildInputs = [
     gtk2
     gtk3
+    gtk4
   ];
```

Comment:
Not sure wether we should still support gtk2.

Reply:
Yes I know, but the idea of ​​awf (original or fork) is to allow updating themes from gtk2 to gtk3.
With both gtk2 and gtk3 builds, you can easily compare the two themes, because windows layout are identical.

Without the gtk2 build, as I said to Debian Reviewers, awf is.... almost useless, mainly because gtk3 provide *gtk3-widget-factory* (and gtk4 provide *gtk4-widget-factory*).